### PR TITLE
Add Browser Subtitle Support To DAOControllerConfig

### PR DESCRIPTION
**Pull request by tharmaman**
Currently is opt-out where the user can pass an empty string in the config to render no subtitle at all or a customized string to render a subtitle that differs from the generic one (View all xyzs)

### CODE REVIEW

File: src/foam/comics/v2/DAOBrowseControllerView.js [hunk 1]
```diff
@@ -115,18 +122,27 @@ foam.CLASS({
...
+              .start(self.Rows)
+                .addClass(self.myClass('header-container'))
+                .start(self.Cols)
+                  .start()
+                    .addClass(self.myClass('browse-title'))
+                    .add(config$browseTitle)
+                  .end()
+                  .startContext({ data: self }).tag(self.CREATE).endContext()
+                .end()
+                .callIf(config$browseTitle.length > 0, function() {
```
Comment (nanoscott) [inline src/foam/comics/v2/DAOBrowseControllerView.js]: this should be checking browserSubtitle, not browserTitle.

**Comment by kgrgreer** [comment]
LGTM except for Scott's comment.

File: src/foam/comics/v2/DAOBrowseControllerView.js (hunk 1 above)
Comment (tharmaman) [reply src/foam/comics/v2/DAOBrowseControllerView.js]: Whoops just pushed fix.

**Comment by tharmaman** [comment]
> LGTM except for Scott's comment.

Just pushed fix to scott's comment

**Review by nanoscott — APPROVED** [review]

File: src/foam/comics/v2/DAOControllerConfig.js [hunk 2]
```diff
@@ -56,6 +56,11 @@ foam.CLASS({
       name: 'browseTitle',
       expression: function(of) { return foam.String.pluralize(of.model_.label); }
     },
+    {
+      class: 'String',
+      name: 'browseSubtitle',
+      expression: function(of) { return "View all " + foam.String.pluralize(of.model_.label.toLowerCase()) +'.' }
```
Comment (kgrgreer) [inline src/foam/comics/v2/DAOControllerConfig.js]: There is a 'plural' property on Model, which you could/should use instead.

Comment (tharmaman) [reply src/foam/comics/v2/DAOControllerConfig.js]: Fixed.

**Pull request merged**